src/net/sourceforge/kolmafia/persistence/EquipmentDatabase.java

```diff
@@ -31,23 +32,21 @@
 import net.sourceforge.kolmafia.preferences.Preferences;
 import net.sourceforge.kolmafia.session.EquipmentManager;
 import net.sourceforge.kolmafia.utilities.FileUtilities;
-import net.sourceforge.kolmafia.utilities.IntegerArray;
 import net.sourceforge.kolmafia.utilities.LogStream;
-import net.sourceforge.kolmafia.utilities.StringArray;
 import net.sourceforge.kolmafia.utilities.StringUtilities;
 
 public class EquipmentDatabase {
-  private static final IntegerArray power = new IntegerArray();
-  private static final IntegerArray hands = new IntegerArray();
-  private static final StringArray itemTypes = new StringArray();
-  private static final StringArray statRequirements = new StringArray();
+  private static final Map<Integer, Integer> power = new LinkedHashMap<>();
+  private static final Map<Integer, Integer> hands = new LinkedHashMap<>();
+  private static final Map<Integer, String> itemTypes = new LinkedHashMap<>();
+  private static final Map<Integer, String> statRequirements = new LinkedHashMap<>();
```

Comment:
Why are these not ArrayLists or HashMaps? They are accessed via .get() and .set(). They are never iterated.